Runner.php

```diff
@@ -52,29 +47,34 @@ public function queuePaths($dir, $prefix = '', $exclusions = []) {
 
             if ($f !== '.' and $f !== '..') {
                 if (is_dir("$dir/$f")) {
-                    $this->queuePaths("$dir/$f", "$prefix$f/", $exclusions);
+                    $this->queuePaths("$dir/$f", "$prefix$f/");
                     continue;
-                }
-
-                if (isset($this->config['config']['file_extensions'])) {
-                    $this->filterByExtension($f, $prefix);
                 } else {
-                    $prefix = ltrim($prefix, "\\/");
-                    $this->server->addwork(array("/code/$prefix$f"));
+                    $this->filterByExtension($f, $prefix);
                 }
             }
         }
     }
 
     public function filterByExtension($f, $prefix = '') {
-        foreach (explode(",", $this->config['config']['file_extensions']) as $file_extension) {
-            if (S::create($f)->endsWith("." . $file_extension)) {
+        foreach ($this->fileExtensions() as $file_extension) {
+            if (S::create($f)->endsWith($file_extension)) {
```

Comment:
The reasoning behind is so that when users provide file extensions we don't care if they give us `.php` or just `php`.